### PR TITLE
general fixes from dogfooding feedback 

### PR DESCRIPTION
**Pull request by paulbalaji**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
update setup for when VS installed already
#### Tests
local
#### Documentation
this is a doc
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/setup-and-installing.md [hunk 1]
```diff
@@ -27,27 +27,32 @@ Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
...
 
-If you are using Visual Studio, within the Visual Studio Installer, on the **Workloads** tab:
+Once you have installed [Visual Studio](https://www.visualstudio.com/downloads/), find the **Workloads** tab in the Visual Studio Installer by clicking on the **More** option for *Visual Studio Build Tools*, and selecting **Modify** from the drop-down menu.
 
-- Select **Game development with Unity**
-- Select **.NET Core cross-platform development**
-- In the **Summary** on the right:
-  - Deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.8)
-  - Make sure **Visual Studio Tools for Unity** is selected
+![]({{assetRoot}}assets/setup/windows/visualstudioworkloads.png)
```
Comment (firtina-improbable) [inline docs/setup-and-installing.md]: nit: missing alt-text

File: docs/welcome.md [hunk 2]
```diff
@@ -57,6 +57,8 @@ You’ll then be ready to learn how to build your own features. Our tutorial for
 
 From your Unity Engine file browser, navigate to where you downloaded the FPS starter project and open the `workers/unity` directory inside the project to get started.
 
+<%(Callout type="info" message="Opening a Unity project for the first time may take while, as Unity needs to import and process the project's assets. Why not make yourself a cup of tea while you wait?")%>
```
Comment (firtina-improbable) [inline docs/welcome.md]: Do we do improbadoc syntax on master?

Comment (paulbalaji) [reply docs/welcome.md]: this is going into the conversion branch, so I assume that is fine?

Comment (firtina-improbable) [reply docs/welcome.md]: ooooooooooooooooooooooooh.

File: docs/welcome.md [hunk 3]
```diff
@@ -123,12 +125,11 @@ An [assembly](https://docs.improbable.io/reference/latest/shared/glossary#assemb
 
 To run a deployment in the cloud, you must upload the worker assemblies to your SpatialOS project. This can only be done in a terminal, via the [spatial CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). You must also give the worker assemblies a name so that you can reference them when launching a deployment.
 
-Using a terminal of your choice - for example, the Command Prompt for Windows - navigate to `gdk-for-unity-fps-starter-project/` and run `spatial cloud upload <assembly_name>`, where `assembly_name` is a name of your choice (e.g fps-assembly). A valid upload command would look like this:
+Using a terminal of your choice - for example, the Command Prompt for Windows - navigate to `gdk-for-unity-fps-starter-project/` and run `spatial cloud upload <assembly_name>`, where `<assembly_name>` is a name of your choice (e.g fps-assembly). A valid upload command would look like this:
```
Comment (jamiebrynes7) [inline docs/welcome.md]: Pleaseeeeeeeeeeeeeeeeeeeeeeeeeeeeeee don't tell people to  use Command Prompt

File: docs/setup-and-installing.md [hunk 4]
```diff
@@ -27,27 +27,32 @@ Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
...
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables
 
 **Step 4.** Install a **code editor** if you don't have one already
 
 - We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
 
 **Using Visual Studio?**
 
-If you are using Visual Studio, within the Visual Studio Installer, on the **Workloads** tab:
+As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find this by clicking on the **More** option for *Visual Studio Build Tools* in the Visual Studio Installer, and selecting **Modify** from the drop-down menu.
```
Comment (paulbalaji) [inline docs/setup-and-installing.md]: @jamiebrynes7 how does this flow?

Comment (jamiebrynes7) [reply docs/setup-and-installing.md]: 👌 Very nice

**Pull request merged**